tests/unit/models/gpflux/conftest.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 The Trieste Contributors
```

Comment:
same as for integration test, for usability lets move fixtures to a conftest in models directory